src/Game/Command/RemindCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php namespace Slackwolf\Game\Command;
+
+use Exception;
+use Slack\Channel;
+use Slack\ChannelInterface;
+use Slack\DirectMessageChannel;
+use Slackwolf\Game\Game;
+
+/**
+ * Defines the RemindCommand class.
+ */
+class RemindCommand extends Command
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fire()
+    {
+        $client = $this->client;
+
+        if ($this->channel[0] == 'D') {
+            $client->getDMByUserId($this->userId)
+                ->then(function(DirectMessageChannel $dm) use ($client) {
+                    $client->send(":warning: Run this command in the game channel.", $dm);
+                });
+            return;
+        }
```

Comment:
Do you think it needs to be public information to the channel that you have forgotten your role? I'm not sure.